charts/backstage/ci/with-custom-image-for-test-pod-values.yaml

```diff
@@ -10,5 +10,5 @@ upstream:
 test:
   image:
     registry: quay.io
-    repository: curl/curl-base
-    tag: 8.11.1
+    repository: rhdh-community/curl
```

Comment:
The point of this file is to test an image other than what we have in the default values.yaml, to make sure that it is possible to configure the image.
Is it possible to use a different image (just for CI) that would meet the new security requirements?